addons/website_blog/models/website_blog.py

```diff
@@ -131,28 +131,20 @@ def _compute_ranking(self):
 
     def _default_content(self):
         return '''
-            <section class="s_text_block">
-                <div class="container">
-                    <div class="row">
-                        <div class="col-lg-12 mb16 mt16">
-                            <p class="o_default_snippet_text">''' + _("Start writing here...") + '''</p>
-                        </div>
-                    </div>
-                </div>
-            </section>
+            <p>''' + _("Start writing here...") + '''</p>
         '''
-
     name = fields.Char('Title', required=True, translate=True, default='')
     subtitle = fields.Char('Sub Title', translate=True)
     author_id = fields.Many2one('res.partner', 'Author', default=lambda self: self.env.user.partner_id)
     active = fields.Boolean('Active', default=True)
     cover_properties = fields.Text(
         'Cover Properties',
-        default='{"background-image": "none", "background-color": "oe_black", "opacity": "0.2", "resize_class": ""}')
+        default='{"background-image": "none", "background-color": "oe_black", "opacity": "0.2", "resize_class": "cover_mid"}')
     blog_id = fields.Many2one('blog.blog', 'Blog', required=True, ondelete='cascade')
     tag_ids = fields.Many2many('blog.tag', string='Tags')
     content = fields.Html('Content', default=_default_content, translate=html_translate, sanitize=False)
     teaser = fields.Text('Teaser', compute='_compute_teaser', inverse='_set_teaser')
+    teaser_short = fields.Text('Teaser Short', compute='_compute_teaser_short', inverse='_set_teaser_short')
```

Comment:
after discussion with qsm, use css is not better.

But I confirm that have 2 fields of teaser is not usefull or at least only for design in demo.
If the writer add a teaser manually (and so that the blog / seo is important for him, and not just a small demo) it will be the same at both place... so it should be beautiful with your layout whatever the size... Can you check to find another solution plz. Thx.

Reply:
Yep, I don't like the CSS solution neither... and the user-case in which a "writer add a teaser manually" is a big issue, I agree.

The only solution I can think of is to keep only one field and use an intermediate value (~200 char)... hopefully it will be ok. I'll let you know

FYI, in attach the current fields in use

![Screenshot 2019-08-16 at 15 40 40](https://user-images.githubusercontent.com/8558351/63171992-fc998100-c03c-11e9-98db-998fbb5004a2.png)

Reply:
@JKE-be , @qsm-odoo . 'Fixed" restoring only one field with an intermediate value of 200 chars. Not exactly what I was aiming for, but it look fine 👍